react-search-component/my-app/src/SearchableList.js

```diff
@@ -0,0 +1,42 @@
+import { useState } from 'react';
+
+export default function SearchableList({ quotes }) {
+  const [filterText, setFilterText] = useState('');
+
+  return (
+    <div className="fate">
+      <SearchBar filterText={filterText} onFilterChange={setFilterText} />
+      <QuoteList quotes={quotes} filterText={filterText} />
+    </div>
+  );
+}
+
+function SearchBar({ filterText, onFilterChange }) {
+  return (
+    <form>
```

Comment:
There's no need for a `form` element here, since there is no form that is being submitted. You are handling the input with code instead.